pkgs/misc/emulators/wine/sources.nix

```diff
@@ -65,17 +65,12 @@ in rec {
   staging = fetchFromGitHub rec {
     # https://github.com/wine-staging/wine-staging/releases
     inherit (unstable) version;
-    sha256 = "sha256-g0NmiypafOAmKDRoRf4uz5NnhFo6uga0fKYNCF29jbE=";
+    sha256 = "sha256-nqXgJe2i1xDW1aCAmwFPshiUI2PtZ/S7p8Cq2ODMTQk=";
     owner = "wine-staging";
     repo = "wine-staging";
-    #rev = "v${version}";
-    # FIXME: replace with line above with 6.10 release
-    # Fix https://bugs.winehq.org/show_bug.cgi?id=51172
-    rev = "5bbe3e47a559b3c04bc8791e0b398a271c772af7";
+    rev = "v${version}";
 
-    # Actually only "d3d11-Deferred_Context" cause problems, two others only dependencies
-    # see FIXME above
-    disabledPatchsets = [ "d3d11-Deferred_Context" "wined3d-CSMT_Main" "nvapi-Stub_DLL" "nvcuvid-CUDA_Video_Support" "nvencodeapi-Video_Encoder" ];
+     disabledPatchsets = [ ];
```

Comment:
Super minor nit, the indent seems to be wrong here.

Reply:
Done